config/locales/devise.pt.yml

```diff
@@ -1,6 +1,6 @@
 # Additional translations at https://github.com/plataformatec/devise/wiki/I18n
 
-en:
+pt:
```

Comment:
@frmendes Because the i18 default locale is set to `:pt` in `config/application.rb`. Should we change it to `:en` and just roll with it?